src/Test/Utilities/Portable/Syntax/TokenUtilities.cs

```diff
@@ -42,7 +42,7 @@ public static class TokenUtilities
                         }
                     }
 
-                    AssertEx.Fail($"Unexpected token.  Actual '{actualAll}' Expected '{expectedAll}'");
+                    AssertEx.Fail($"Unexpected token.  Actual '{actualAll}' Expected '{expectedAll}'\r\nActual:\r\n{actual}");
```

Comment:
So there are two actuals? This seems confusing.

Reply:
@jasonmalinowski Does that make sense, or should I change it?

Reply:
@jasonmalinowski Are you ok with this test output, despite "Actual" appearing multiple times?

Reply:
@jcouv I'm still completely unable to explain what it means. Why do we need two?

Reply:
@jasonmalinowski The existing assert only prints a few tokens. This is convenient when you have a small difference (it pinpoints the difference). 
But when you have a large difference (let's say you're writing a new test), that is not very useful. The message I added is useful then. 

I don't think the printout is confusing. Let me know what you want to do: change labels (use `"Actual (full):"`), remove the new section, or something else entirely ;-)

Reply:
For context, see the example I copied above.

Reply:
Latest commit removes some waste. Here's what the new output looks like:

```
Test 'Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings.ExtractMethod.ExtractMethodTests.TestDeconstruction5' failed:
	Unexpected token.  Actual 'y ) ; ^System^ . Console . Write ( ' Expected 'y ) ; ^}^ private static void NewMethod ( '
Actual:
class Program
{
    void M()
    {
        int x, y;
        NewMethod(out x, out y);
        System.Console.Write(x + y);
    }

    private static void NewMethod(out int x, out int y)
    {
        (x, y) = (1, 2);
    }
}
	at Roslyn.Test.Utilities.AssertEx.Fail(String message)
	at Roslyn.Test.Utilities.TokenUtilities.AssertTokensEqual(String expected, String actual, String language)
	at Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions.AbstractCodeActionOrUserDiagnosticTest.<TestOperationsAsync>d__24.MoveNext()
	--- End of stack trace from previous location where exception was thrown ---
	at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
    ....
```

Reply:
@jasonmalinowski Good to go?

Reply:
Oh sorry, yeah, ship it.